test/specs/templates.e2e.ts

```diff
@@ -671,7 +680,12 @@ describe('Templates', async () => {
     expect(outputPanelText).not.toBeUndefined();
 
     // Check for expected item in the Explorer view.
-    testItem = await treeViewSection.findItem('lightningWebComponent1.test.js');
+    const sidebar = workbench.getSideBar();
+    const treeViewSection = await sidebar.getContent().getSection(projectName);
```

Comment:
If this is a common pattern open sidebar for project, expand, find down the tree, then how can this be optimized for brevity?